pkg/controller/jobframework/reconciler.go

```diff
@@ -607,8 +613,39 @@ func (r *JobReconciler) ensurePrebuiltWorkloadInSync(ctx context.Context, wl *ku
 	return true, nil
 }
 
+// get the expected podsets during the job execution, returns nil if the workload has no reservation or
+// the admission dose not fit.
```

Comment:
```suggestion
// the admission does not fit.
```

did you mean match instead of fit?